frontend/appflowy_flutter/lib/user/presentation/screens/sign_in_screen/sign_in_screen.dart

```diff
@@ -53,7 +59,7 @@ class SignInScreen extends StatelessWidget {
         if (user.encryptionType == EncryptionTypePB.Symmetric) {
           router.pushEncryptionScreen(context, user);
         } else {
-          router.pushWorkspaceStartScreen(context, user);
+          router.pushHomeOrWorkspaceStartScreen(context, user);
```

Comment:
@LucasXu0 @appflowy 
I noticed I used the wrong route during the naming changes. It should be the same as the main branch now.